package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/stylelint-processor-arbitrary-tags",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Let's leave this change out of the PR. I'll make the change when I cut a new release.